Spatial.gd

```diff
@@ -57,6 +57,12 @@ const button_margin = {
 	2: {"left": 361, "right": 394}, # large
 	3: {"left": 395, "right": 431}  # larger
 }
+const minimap_scale = {
+	0: {"offset": 32, "factor": 0.904}, # small
+	1: {"offset": 36, "factor": 1}, # normal
+	2: {"offset": 40, "factor": 1.118}, # large
+	3: {"offset": 44, "factor": 1.223} # larger
```

Comment:
Where did the `factor` values come from?

Reply:
When doing the research for my other PR #225 I learned that the interface is always generated for the ui-scale `normal` and then rescaled before it gets drawn on the screen. I found that every single button and consequently the menu as a whole was rescaled with a constant factor with `normal` as base-value. So I used the values for the left margin of the burrito button to define those factors because this value also represents the total length of the main menu.

```
small: 292/323=0.904
normal: 323/323=1
large: 361/323=1.118
larger: 395/323=1.223
```

Reply:
Seems reasonable. But they seem so close to 10%/11% increments which would cause me to want to investigate further. Given that this pr is for the minimap, do you get the same four ratios when you count the pixels that the minimap takes up in each uisize? If it were me I would also want to check some of the non resizable windows and the skill bar too.

Reply:
Every ui element is rescaled by the same factor.
After having a closer look I think it is safe to say that the scale is `90%` for `small`, `111%` for `large` and `122%` for `larger`. 

![scale_minimap](https://github.com/AsherGlick/Burrito/assets/30129213/3fe4e1a0-d765-48af-80d4-87f8a45fe645)
![scale_skillbar](https://github.com/AsherGlick/Burrito/assets/30129213/1495bb35-4a49-49ee-a447-97d2e60e05d9)

Reply:
```suggestion
	0: {"offset": 32, "factor": 0.9}, # small
	1: {"offset": 36, "factor": 1}, # normal
	2: {"offset": 40, "factor": 1.11}, # large
	3: {"offset": 44, "factor": 1.22} # larger
```

Reply:
:white_check_mark: